src/main/java/appeng/parts/automation/PartLevelEmitter.java

```diff
@@ -206,6 +206,7 @@ public void updateWatcher( ICraftingWatcher newWatcher )
 	@Override
 	public void onRequestChange( ICraftingGrid craftingGrid, IAEItemStack what )
 	{
+		System.out.println(what);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)